.github/workflows/test.yml

```diff
@@ -0,0 +1,60 @@
+name: test
+
+on:
+  push:
+    branches: main
+  pull_request:
+    branches: main
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.9"
+      - name: Install Poetry
+        run: python -m pip install --user poetry
+      - name: Install dependencies
+        run: poetry install --no-interaction --no-root
+      - name: Format with isort
+        run: poetry run isort .
+      - name: Format with black
+        run: poetry run black .
+      - name: Commit changes if any
+        run: |
+          git config user.name "GitHub Actions"
+          git config user.email "action@github.com"
+          if output=$(git status --porcelain) && [ ! -z "$output" ]; then
+            git commit -m "style: Automatic code formatting" -a
+            git push
+          fi
```

Comment:
It would feel a bit safer if CI didn't automatically commit changes to the code, as unexpected things might happen. So, I'd prefer someone to verify the errors and manually fix them.

Would it be possible to warn the developer rather than commit the changes?

Reply:
Changed by https://github.com/GeriLife/companionship-care/pull/61/commits/38e8facd158db857ddcbe19b4601cc84a7773fef